COLLABORATOR_GUIDE.md

```diff
@@ -411,64 +411,45 @@ recommended but not required.
-deprecation as a semver-minor.
-
-All Documentation-Only and Runtime deprecations will be assigned a unique
-identifier that can be used to persistently refer to the deprecation in
-documentation, emitted process warnings, or errors thrown. Documentation for
-these identifiers will be included in the Node.js API documentation and will
-be immutable once assigned. Even if End-of-Life code is removed from Node.js,
-the documentation for the assigned deprecation identifier must remain in the
-Node.js API documentation.
+* *Documentation-Only Deprecation*: A deprecation notice is added to the API
+  documentation but no functional changes are implemented in the code. There
+  will be no runtime deprecation warnings emitted for such deprecations by
+  default. (They may trigger a runtime warning when used with the
+  [`--pending-deprecation`][] CLI option or the `NODE_PENDING_DEPRECATION`
+  environment variable.)
+
+* *Runtime Deprecation*: A warning is emitted at runtime the first time that the
+  deprecated API is used. The [`--throw-deprecation`][] CLI option can be used
+  to treat these warnings as runtime errors. Documentation for the API must
+  indicate that the deprecation is a Runtime Deprecation.
+
+* *End-of-life*: The API is no longer subject to semantic versioning rules.
+  Backward-incompatible changes (including removal of the API) may occur at any
+  time. Documentation for the API must indicate that the API is End-of-life.
+
+The introduction of a Documentation-Only Deprecation is a semver-minor change.
+
+Both the introduction of a Runtime Deprecation and the subsequent move of an API
+to End-of-life are semver-major.
+
+Deprecations are assigned a unique and persistent identifier for use in
+documentation, warnings, and errors. All such identifiers are documented. They
+may not be changed or removed from the documentation, even after the relevant
+API has been removed from Node.js.
 
 <a id="deprecation-cycle"></a>
 A _Deprecation cycle_ is one full Node.js major release during which an API
-has been in one of the three Deprecation levels. (Note that Documentation-Only
-Deprecations may land in a Node.js minor release but must not be upgraded to
-a Runtime Deprecation until the next major release.)
+has been in one of the three Deprecation levels.
 
 No API can be moved to End-of-life without first having gone through a
 Runtime Deprecation cycle. However, there is no requirement that deprecated
 code must progress ultimately to *End-of-Life*. Documentation-only and runtime
 deprecations may remain indefinitely.
 
-A best effort will be made to communicate pending deprecations and associated
-mitigations with the ecosystem as soon as possible (preferably before the pull
-request adding the deprecation lands on the master branch). All deprecations
-included in a Node.js release should be listed prominently in the "Notable
-Changes" section of the release notes.
```

Comment:
I think it is good to keep it as as. This makes it clear to the reader that we try to report everything we find but we can not always handle everything. It also helps collaborators to know that this is something they should do when opening a PR.

Reply:
For me the first sentence seems pretty important. I understand the `something should be done` part as a request to check CITGM and to open PRs in userland modules to fix the upcoming issues. This is done with a "best effort" and we can not guarantee anything and decide if it makes sense to do it or not.